src/coinbase/wallet.ts

```diff
@@ -89,6 +89,28 @@ export class Wallet {
     return Wallet.init(response.data!, "");
   }
 
+  /**
+   * Imports a Wallet for the given Wallet data.
+   *
+   * @param data - The Wallet data to import.
+   * @returns The imported Wallet.
+   * @throws {ArgumentError} If the Wallet ID is not provided.
+   * @throws {ArgumentError} If the seed is not provided.
+   * @throws {APIError} If the request fails.
+   */
+  public static async import(data: WalletData): Promise<Wallet> {
+    if (!data.walletId) {
+      throw new ArgumentError("Wallet ID must be provided");
+    }
+    if (!data.seed) {
+      throw new ArgumentError("Seed must be provided");
+    }
+    const walletModel = await Coinbase.apiClients.wallet!.getWallet(data.walletId);
+    const wallet = Wallet.init(walletModel.data, data.seed);
+    await wallet.listAddresses();
```

Comment:
we dont seem to be listing addresses here https://github.com/coinbase/coinbase-sdk-ruby/blob/master/lib/coinbase/wallet.rb#L38